src/scenes/Question/scenes/Edit/components/Tips/Tips.jsx

```diff
@@ -1,16 +1,93 @@
 import React from 'react'
 
-import Card, { CardText } from 'components/Card'
+import { default as TipsComponent } from 'components/Tips'
 
 const Tips = props => (
-  <Card {...props}>
-    <CardText>
-      <h3 style={{ textAlign: 'center', color: 'var(--primary-color)' }}>
-        Tips to write great questions on the FAQ
-      </h3>
-      --Need help--
-    </CardText>
-  </Card>
+  <TipsComponent {...props} uid="tips_question">
+    <h3>Tips to write great questions on the FAQ</h3>
+    <div style={{ textAlign: 'center' }}>
+      <ul style={{ textAlign: 'left' }}>
+        <li>
+          Do some searches before you can be sure your question has not already
+          been asked.
+        </li>
+        <li>Be concise and precise in your wording.</li>
+        <li>
+          <span>
+            Privilege questions responding to the <b>how</b> rather than the{' '}
+            <b>why</b>.
+          </span>
+        </li>
+        <li>If the wording is not perfect, someone else can improve it.</li>
+        <li>Stay objective and factual.</li>
+      </ul>
+    </div>
+    <div className="row-on-mobile">
+      <div>
+        <h3>
+          <i className="material-icons" style={{ color: 'green' }}>
+            check
+          </i>
+          Great questions
+        </h3>
+        <ul>
+          <li>Where can I get the ElasticSearch training documents?</li>
```

Comment:
I think "material" is a more idiomatic term than "document" in this case, though I'm not 100% sure. A quick Google search seem to agree.